src/lib/GameList/GameListCarousel.svelte

```diff
@@ -0,0 +1,112 @@
+<script lang="ts">
+	import type { Game } from '$lib/games';
+	export let gamesData: Game[];
+	export let carouselTitle;
+
+	// export let game: Game;
+
+	// Target Group Functionality
+	// const targetGroupArr = game.targetGroup;
+	// targetGroupArr.sort(function (a, b) {
+	// 	return a - b;
+	// });
+	// const minGroup = targetGroupArr[0];
+	// const maxGroup = targetGroupArr.slice(-1)[0];
+	// const targetGroupString = `Groep ${minGroup} t/m ${maxGroup}`;
+</script>
+
+<section>
+	<h2>{carouselTitle} - {gamesData.length}</h2>
+	<ul>
+		{#each gamesData as game}
+			<a href="spellen/{game.slug}">
+				<h1>{game.name}</h1>
+				<div>
+					<p class="label">Groep: {game.targetGroup}</p>
+					<p class="label">{game.category}</p>
+					<p class="label">Min. {game.minimumPlayers} spelers</p>
+				</div>
```

Comment:
Hier kan je gewoon de het component `GameListCard` gebruiken. Dit houd de boel een beetje consistent als er een wijziging is in bijvoorbeeld de labels